modules/rosa-configuring-autorepair.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+// * nodes/rosa-managing-worker-nodes.adoc
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-configuring-autorepair_{context}"]
+= Configuring machine pool AutoRepair
+
+{hcp-title} supports an automatic repair process for machine pools. AutoRepair is useful when you would want the ROSA service to detect certain unhealthy machines, drain the unhealthy nodes, and re-create the nodes. You can disable auto-repair if the unhealthy machines should not be replaced in cases where the nodes or the unhealthy instances should be preserved. AutoRepair is enabled by default on Machine Pools. 
```

Comment:
automatic repair, AutoRepair, auto-repair. 
Should we settle on one? 
```suggestion
{hcp-title} supports an automatic repair process for machine pools, called AutoRepair. AutoRepair is useful when you want the ROSA service to detect certain unhealthy machines, drain the unhealthy nodes, and re-create the nodes. You can disable AutoRepair if the unhealthy machines should not be replaced, such as in cases where the nodes or the unhealthy instances should be preserved. AutoRepair is enabled by default on Machine Pools. 
```
Also, machines, nodes, instances. Are these three different concepts, or different words for the same thing?

Reply:
Same thing, I will make it consistent